Mathlib/Topology/MetricSpace/Isometry.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Sébastien Gouëzel
 -/
 import Mathlib.Topology.MetricSpace.Antilipschitz
+import Mathlib.Data.Fintype.Lattice
```

Comment:
I'm surprised you need to import this here. Is it not already transitively imported?
```suggestion
```

Reply:
I thought so, too. But lean does not find `Finset.sup_univ_eq_iSup` if I take the import away.